### ISSUE TITLE
The format of vulnerability scanning result report should be refactored and improved to avoid confusions.

### ISSUE THREAD

**Issue by steven-zou** [vmware/harbor#4567]
Currently, we summarize info based on package. Maybe it should be displayed directly with CVE counts of different severities.

**Comment by zhoumeina** [comment]
I will asign this UI changes to the new comer to familiar with the code.

**Comment by zhoumeina** [comment]
@lweitzman , I agree. It do lead misunderstanding of Vulnerabilities number. Could you help to give a new design about this? What about show "XX Vulnerabilities in XX packages."
